applications/ParticleMechanicsApplication/custom_strategies/schemes/mpm_residual_based_bossak_scheme.hpp

```diff
@@ -534,6 +546,17 @@ class MPMResidualBasedBossakScheme
     unsigned int mBlockSize;
     MPMBoundaryRotationUtility<LocalSystemMatrixType,LocalSystemVectorType> mRotationTool;
 
+    void ClearReaction() const
+    {
+        #pragma omp parallel for
+        for (int iter = 0; iter < static_cast<int>(mGridModelPart.Nodes().size()); ++iter) {
+            auto i = mGridModelPart.NodesBegin() + iter;
+            (i)->SetLock();
+            (i)->FastGetSolutionStepValue(REACTION).clear();
+            (i)->UnSetLock();
```

Comment:
I don't see why we need `SetLock` and `UnSetLock` here

Reply:
I agree in this case its superfluous. Thanks for the review